Automattic/wp-calypso
# Site Editor Domain Upsell: Add callout (no iframe)

Fixes https://github.com/Automattic/wp-calypso/issues/76321
Fixes https://github.com/Automattic/wp-calypso/issues/73445

## Proposed Changes

Add Domain Upsell callout to Gutenberg using [wpcom block editor](https://github.com/Automattic/wp-calypso/blob/trunk/apps/wpcom-block-editor/README.md)

## Testing Instructions

- Follow [this instructions](https://github.com/Automattic/wp-calypso/tree/trunk/apps/wpcom-block-editor#build) to run `wpcom-editor-toolkit` on your sandbox
- You have to sandbox `widgets.wp.com`
- On Calypso, go to `/posts` (and `/pages`) and edit or create new.
- Or, change the view to classic and use `/wp-admin`
- You should see the callout.
- Test clicking on the CTA.
- Test dismissing it.
- If you dismiss it, check on another browser if the dismiss persists.

> **Note**
> To re-enable the callout after you dismissed it, you have to run `wp.data.dispatch( 'core/preferences' ).set( 'core/edit-post', 'domain_upsell_callout_dismiss', false );` in the console and then reload the editor


#### Note: 

#### Pre-merge Checklist





Related to #
